### PR TITLE
Adds blog post about release candidates

### PR DESCRIPTION
**Pull request by stockholmux**
### Description

A short blog post about why/who/how of Valkey release candidates.
 
### Issues Resolved

n/a

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by madolson — CHANGES_REQUESTED** [review]
Mostly minor comments and some types. Overall I think the blog is good!

File: content/blog/2026-04-09-on-release-candidates/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+server_name:valkey
+valkey_version:9.1.0
+valkey_release_stage:rc1
+```
+
+## Feedback and next steps
+
+When you test a release candidate, it’s important to report bugs promptly. While each individual feature is tested, your unique environment and use-case may reveal corner cases that are yet unknown. The best way to report feedback is via an issue on GitHub. Make sure when you report issues you clearly indicate that you’re using a release candidate.
+
+While the project values feedback of all forms, if you find out during a release candidate that you don’t like or agree with a new API, return values, or if you think something is missing or doesn’t belong, it *may be too late* to have any real impact. While not impossible, it would be unlikely for a release candidate to trigger a large change in these areas. Primarily release candidates help validate stability, compatibility, and performance.
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply content/blog/2026-04-09-on-release-candidates/index.md]: Okay. I'd still like to keep the bit about what they are really for though (stability etc.). I'll add it back in.

**Comment by stockholmux** [comment]
Huh. DCO Bot is unhappy with my co-authored commits but fine with my solo authored commits using the same sign off. 🧐 

I'm setting it to pass because they are legit signed off.

**Review by madolson — APPROVED** [review]
Here is the stamp for when you need it.

File: content/blog/2026-04-28-on-release-candidates/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
++++
+title = "On release candidates"
+date = 2026-04-09
+description = "A release candidate just came out for Valkey. You’ve likely heard of the term before but do you know what it really means?"
+authors = ["kyledvs"]
+[extra]
+featured = true
```
Comment (madolson) [inline content/blog/2026-04-28-on-release-candidates/index.md:9]: Need the new tag
